src/2dplane/GridStateSpace.hpp

```diff
@@ -19,6 +19,9 @@ class GridStateSpace : public PlaneStateSpace {
     bool stateValid(const Eigen::Vector2f &pt) const;
     bool transitionValid(const Eigen::Vector2f &from, const Eigen::Vector2f &to) const;
 
+    //a negative stepSize indicates that adaptive stepsize control is being used
+    Eigen::Vector2f intermediateState(const Eigen::Vector2f &source, const Eigen::Vector2f &target, float stepSize, float limit) const;
```

Comment:
Can we rename the last two params to `minStepSize` and `maxStepSize` to be a little clearer about what they're doing?  Also, the comment above doesn't apply anymore, does it?